src/types/List.js

```diff
@@ -0,0 +1,38 @@
+import { extend } from '../utils/adopter.js'
+import { subClassArray } from './ArrayPolyfill.js'
+
+const List = subClassArray('List', Array, function (arr) {
+  this.length = 0
+  this.push(...arr)
+})
+
+export default List
+
+extend(List, {
+  each (cbOrName, ...args) {
+    if (typeof cbOrName === 'function') {
+      this.forEach((el) => { cbOrName.call(el, el) })
```

Comment:
What is cbOrName? Give me a better variable name here :P

Reply:
Its a callback or a name - what do you want more? :D 
Okeee - maybe `cbOrMethodName`